maps-compose/build.gradle

```diff
@@ -36,9 +36,11 @@ android {
 
 dependencies {
     implementation "androidx.compose.foundation:foundation:$compose_version"
+    implementation 'androidx.compose.material:material:1.1.1'
     implementation 'androidx.core:core-ktx:1.7.0'
     implementation 'com.google.android.gms:play-services-maps:18.0.2'
-    implementation 'com.google.maps.android:maps-ktx:3.3.0'
+    implementation 'com.google.maps.android:maps-ktx:3.4.0'
+    implementation 'com.google.maps.android:maps-utils-ktx:3.3.0'
```

Comment:
Yeah, that makes a lot of sense to me. I like "widgets" as well, so `maps-compose-widgets` sounds good to me!

Reply:
Yep, you can duplicate what's in there. Publishing code is on the project-level `build.gradle` file. You will have to make a new entry for `artifactId` so that it's applied to the new module. See: https://github.com/googlemaps/android-maps-compose/blob/main/build.gradle#L24

Reply:
Thanks! Done in https://github.com/googlemaps/android-maps-compose/pull/157/commits/32e5a4e4350774c3c7bca05dce23d2b907617e6e. Eyes on this are welcome to make sure I didn't miss anything.